prefix-hex/src/primitive_types.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use alloc::{format, string::String};
+
+use crate::{strip_prefix, Error, FromHexPrefix, ToHexPrefix};
+
+#[rustfmt::skip]
+fn is_hex_character(c: char) -> bool {
+    matches!(c, '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9' |
+                'a' | 'b' | 'c' | 'd' | 'e' | 'f' |
+                'A' | 'B' | 'C' | 'D' | 'E' | 'F')
+}
```

Comment:
Can we not use:
https://doc.rust-lang.org/core/primitive.char.html#method.is_ascii_hexdigit ?

Reply:
Nice find!